### PR TITLE
Adding hello-eks-anywhere for packages testing, moving away from eks-anywhere-test-image

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jonahjon** [comment]
This is for https://github.com/aws/eks-anywhere/issues/1739

**Comment by TerryHowe** [comment]
/hold
it would be nice to have the prow job first

File: projects/aws-containers/hello-eks-anywhere/conf/default.conf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+map $http_user_agent $root {
```
Comment (TerryHowe) [inline projects/aws-containers/hello-eks-anywhere/conf/default.conf:1]: Did we need to change what is in the source repo? https://github.com/aws-containers/hello-eks-anywhere/tree/main/conf

**Pull request closed without merging**